trino/dbapi.py

```diff
@@ -220,6 +221,31 @@ def cursor(self, experimental_python_types: bool = None):
         )
 
 
+class ColumnDescription(NamedTuple):
+    name: str
+    type_code: int
+    display_size: int
+    internal_size: int
+    precision: int
+    scale: int
+    null_ok: bool
+
+    @classmethod
+    def from_column(cls, column: Dict[str, Any]):
+        type_signature = column["typeSignature"]
+        raw_type = type_signature["rawType"]
+        arguments = type_signature["arguments"]
+        return cls(
+            column["name"],  # name
+            raw_type if raw_type in PRECISION_TYPES + LENGTH_TYPES else column["type"],  # type_code
+            None,  # display_size
+            arguments[0]["value"] if len(arguments) >= 1 and raw_type in LENGTH_TYPES else None,  # internal_size
+            arguments[0]["value"] if len(arguments) >= 1 and raw_type in PRECISION_TYPES else None,  # precision
+            arguments[1]["value"] if len(arguments) >= 2 and raw_type in SCALE_TYPES else None,  # scale
```

Comment:
why `len(arguments) >= 2` ?

Reply:
We can drop this all together actually as we check on the types itself already.